src/Akihabara.Tests/Gpu/EglSurfaceHolderTest.cs

```diff
@@ -10,19 +9,20 @@
 
 namespace Akihabara.Tests.Gpu
 {
+    [TestFixture]
     public class EglSurfaceHolderTest
     {
-        #region Constructor
+#region Constructor
```

Comment:
How does this have no indentation here yet is just one level of indentation lower in other places?